source/MaterialXCore/Node.h

```diff
@@ -326,6 +326,13 @@ class MX_CORE_API NodeGraph : public GraphElement
     }
     virtual ~NodeGraph() { }
 
+    /// @name Material References
+    /// @{
+    
+    /// Return any outputs on the nodegraph which are of type "material"
```

Comment:
Following the pattern above, I would phrase this as "Return all material-type outputs of the nodegraph."

Reply:
Will do.